src/commands/economy/CoinflipCommand.js

```diff
@@ -54,8 +54,8 @@ module.exports = class CoinflipCommand extends Command {
           ctx.send(`${ctx.locale('commands:coinflip.coroa')}\n${user2} ${ctx.locale('commands:coinflip.coroa-texto', { value: valor })} ${user1}`);
         }
 
-        await this.client.database.repositories.starRepository.add(winner, valor);
-        await this.client.database.repositories.starRepository.remove(loser, valor);
+        await this.client.repositories.starRepository.add(winner, valor);
+        await this.client.repositories.starRepository.remove(loser, valor);
```

Comment:
transaction. deve ser uma função só
```suggestion
        await this.client.repositories.starRepository.coinflip(winnerID, loserID, value);
```